pkg/sinks/sinker.go

```diff
@@ -19,10 +19,12 @@ package sinks
 import (
 	"github.com/numaproj/numaflow/pkg/forward"
 	"github.com/numaproj/numaflow/pkg/isb"
+	"github.com/numaproj/numaflow/pkg/shared/util"
 )
 
 // Sinker interface defines what a Sink should implement.
 type Sinker interface {
 	isb.BufferWriter
 	forward.StarterStopper
+	util.ServerHandler
```

Comment:
Instead of adding to Sinker interface, which requires all the sinks to implement it, can we do a instance of check in sink.go, then only udsink needs to do it?